test/inductor/test_utils.py

```diff
@@ -203,5 +205,88 @@ def test_get_device_tflops(self, dtype):
 
 instantiate_device_type_tests(TestUtils, globals())
 
+
+class TestFakeTensorUpdater(TestCase):
```

Comment:
I'm wondering if there's an easier to write these tests so that we can write a lot more tests. Each of these tests currently is pretty long unfortunately.

Strawman:
- have an API that accepts a function, the name of the node to change, and a number of updated nodes
- the function will capture a graph with AOTEagerAndRecordGraphs()
- it will delete the val field from said node
- then it will run faketensor updator
- finally it'll assert against the number of updated nodes

The goal is to just have more comprehensive testing